js/completionStateEnum.js

```diff
@@ -0,0 +1,8 @@
+
+const COMPLETION_STATE = ENUM([
+  'INVIEW',
+  'PLAY',
+  'ENDED'
+], value => value?.toUpperCase?.());
+  
+export default COMPLETION_STATE;
```

Comment:
ESLint: missing new line at end of file.
```suggestion
export default COMPLETION_STATE;

```